app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="playlist_confirm_all_duplicates">Deze afspeellijst bevat al deze nummers al. Als u doorgaat zullen er %d dubbele nummers zijn.</string>
+    <string name="confirm_dir_include_excluded">“%1$s” wordt al uitgesloten. Wilt u het insluiten?</string>
+    <string name="confirm_dir_exclude_included">“%1$s” wordt al ingesloten. Wilt u het uitsluiten?</string>
+    <string name="confirm_dir_already_excluded">“%1$s” wordt al uitgesloten</string>
+    <string name="confirm_dir_already_included">“%1$s” wordt al ingesloten</string>
+    <string name="message_removed_directory">Map “%1$s” is verwijderd</string>
+    <string name="nothing_playing">Er wordt niets afgespeeld</string>
+    <string name="unknown_artist">Onbekende Artiest</string>
+    <string name="unknown_album">Onbekend Album</string>
+    <string name="unknown">Onbekend</string>
+    <string name="empty_no_permission">Jockey heeft rechten nodig om uw muziekbibliotheek uit te lezen</string>
+    <string name="empty_no_permission_detail">Om uw muziek te laten zien heeft Jockey rechten nodig tot de externe opslag</string>
+    <string name="error_no_permission_dirs">Om mappen uit te sluiten heeft Jockey toegang nodig tot de externe opslag</string>
+    <string name="empty">Hier is geen muziek te vinden</string>
+    <string name="empty_detail">Je kan muziek synchroniseren vanaf de computer of downloaden vanaf het internet</string>
+    <string name="empty_genres">Kan geen genres vinden</string>
+    <string name="empty_playlists">Je hebt nog geen afspeellijsten</string>
+    <string name="empty_playlist">Deze afspeellijst is leeg</string>
+    <string name="empty_auto_playlist">Deze afspeellijst is leeg</string>
+    <string name="empty_search">Geen resultaten</string>
+    <string name="empty_auto_playlist_detail">Geen nummers die je filter volgden</string>
+    <string name="empty_playlist_detail">Je kan muziek toevoegen in de bibliotheek pagina</string>
+    <string name="empty_playlists_detail">Je kan een nieuwe maken door op de “+” knop te drukken</string>
+    <string name="empty_genres_detail">Zorg ervoor dat je muziek correct is getagd</string>
+    <string name="empty_queue">Er wordt niets afgespeeld</string>
+    <string name="empty_error_album">Kan het album niet laden</string>
+    <string name="empty_error_artist">Kan de artiest niet laden</string>
+    <string name="empty_error_genre">Kan het genre niet laden</string>
+    <string name="empty_error_folder">Kan de map niet laden</string>
+    <string name="color_grey">Grijs</string>
+    <string name="color_red">Rood</string>
+    <string name="color_orange">Oranje</string>
+    <string name="color_yellow">Geel</string>
+    <string name="color_green">Groen</string>
+    <string name="color_cyan">Cyaan</string>
+    <string name="color_blue">Blauw</string>
+    <string name="color_purple">Paars</string>
+    <string name="color_black">Zwart</string>
+    <string name="color_teal">Teal</string>
```

Comment:
Google Translate is saying that this should be translated as "Taling". Is that accurate?

Reply:
"Teal" is not used in the Dutch language, so it is spoken as the English word.

Google Translate is wrong here, a "taling" is a bird/duck and not a color.